WebService/v1/Models/DeviceModelApiModel/DeviceModelApiModel.cs

```diff
@@ -56,14 +56,19 @@ public class DeviceModelApiModel
         public IDictionary<string, string> Metadata => new Dictionary<string, string>
         {
             { "$type", "DeviceModel;" + v1.Version.NUMBER },
-            { "$uri", "/" + v1.Version.PATH + "/devicemodels/" + this.Id },
+            { "$uri", "/" + v1.Version.PATH + "/deviceModels/" + this.Id },
             { "$created", this.created.ToString(DATE_FORMAT) },
             { "$modified", this.modified.ToString(DATE_FORMAT) }
         };
 
+        private const string NO_PROTOCOL = "The device model doesn't contain a protocol";
+        private const string INVALID_PROTOCOL = "The device model has a invalid protocol";
```

Comment:
'The device model has an invalid protocol'

Reply:
Done.